linera-service/src/linera/main.rs

```diff
@@ -1348,7 +1348,10 @@ fn log_file_name_for(command: &ClientCommand) -> Cow<'static, str> {
         ClientCommand::Project { .. } => "project".into(),
         ClientCommand::Watch { .. } => "watch".into(),
         ClientCommand::Storage { .. } => "storage".into(),
-        ClientCommand::Service { port, .. } => format!("service-{port}").into(),
+        ClientCommand::Service {
+            port: Some(port), ..
+        } => format!("service-{port}").into(),
+        ClientCommand::Service { port: None, .. } => "service".into(),
```

Comment:
Is there a danger that multiple services with dynamically assigned ports now use the same log file?

Reply:
Yes, but this only matters when using the `LINERA_LOGS_DIR` environment variable. Long term we should probably update the tests to pass an `--id` to the node services.